client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx

```diff
@@ -46,6 +45,17 @@ const PremiumPlanDetails = ( {
 				hasDomainCredit={ plan && plan.hasDomainCredit }
 			/>
 
+			<PurchaseDetail
+				icon={ <img alt={ translate( 'Earn Illustration' ) } src={ earnImage } /> }
+				title={ translate( 'Make money with your website' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Make money from your website' )` *ES Score: 6*